FirebaseSessions/ProtoSupport/Protos/clientanalytics.proto

```diff
@@ -0,0 +1,67 @@
+
+syntax = "proto2";
+
+// This is in a separate file so it can be used as proto2.
```

Comment:
Question (I didn't look too hard) why is the clientanalytics proto included? Who will use it? I figured the sessions proto will be a field on it but I'm not seeing a field below where that would be the case.

Reply:
We use the `NetworkConnectionInfo` in the Session proto. Nanopb does not like the fully qualified namespace so that may have made it less clear to the eye

Reply:
This does not need to be in a separate proto file. In firebase performance, we packed these enums to the same parent proto to avoid referencing multiple proto files. Can we do something similar here?

https://github.com/firebase/firebase-ios-sdk/tree/master/FirebasePerformance/ProtoSupport/Protos

https://github.com/firebase/firebase-ios-sdk/blob/master/FirebasePerformance/ProtoSupport/Protos/perf_metric.proto#L387-L445

Reply:
This file is proto2, our main proto is proto3. This file won't build as proto3 because it has non-zero defaults (eg. NetworkType.NONE has a value of -1). We might be able to combine these files if we convert our sessions.proto to proto2, but that seems risky if our backend is proto3, and it will be confusing for updating it (because you'll have to remember to swap it from proto3 to proto2).

Fireperf is a different case cause the main proto is proto2